server/cli/start.go

```diff
@@ -41,21 +40,21 @@ var Context = server.NewContext()
 
 // An initCmd command initializes a new Cockroach cluster.
 var initCmd = &commander.Command{
-	UsageLine: "init <storage-location>",
+	UsageLine: "init -stores=<storage-location>[,<storage-location>...]",
```

Comment:
Actually, we should accept the -stores command line flag, not a list of storage locations. change the UsageLine to:

 	UsageLine: "start -stores=(ssd=<data-dir>,hdd:7200rpm=<data-dir>|mem=<capacity-in-bytes>)[,...]",